llvm-hs-pure/src/LLVM/IRBuilder/Module.hs

```diff
@@ -27,7 +28,9 @@ import qualified Control.Monad.State.Strict as Strict
 import Control.Monad.State.Lazy
 import Control.Monad.List
 import Control.Monad.Trans.Maybe
+#if !(MIN_VERSION_mtl(2,2,2))
 import Control.Monad.Trans.Identity
+#endif
```

Comment:
`Control.Monad.Identity` now re-exports `Control.Monad.Trans.Identity` ([v2.2.2](https://github.com/haskell/mtl/blob/master/CHANGELOG.markdown#222)).

It's not that important I guess. I can delete this whole commit.

Reply:
No, this is great! I’ve been annoyed by the warning myself.